.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: test
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.2.0
+      - name: Install Nix
+        uses: cachix/install-nix-action@v18
+        with:
+          extra_nix_config: access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think I need this, but I copied it from the other workflow since I'm not sure what it's being used for.

Reply:
This is needed because otherwise github actions often run into api request limits.